lib/hatchet/app.rb

```diff
@@ -379,7 +387,7 @@ def in_directory_fork(&block)
 
     def deploy(&block)
       in_directory do
-        self.setup!
+        self.in_dir_setup!
```

Comment:
Does the `setup!` in `run_ci` on line 445 need to be changed to `in_dir_setup` as well?

Reply:
Yep! Good catch. It looks like that's why tests were failing.